server/k8s_deploy.sh

```diff
@@ -0,0 +1,23 @@
+## Docker build and push ## 
+# We are inside the `server` directory
+docker build -t gcr.io/${PROJECT_ID}/search_service .
+docker push gcr.io/${PROJECT_ID}/search_service
```

Comment:
@deep-diver this deployment shell-script is supposed by running from a GCP console terminal since it comes preconfigured with all the system-level dependencies. 

Let me know if you run into anything weird.

Reply:
Sure! Let's see how it goes and if we can make github action to deploy automatically

Reply:
Sure thing. Keep me posted.